app/assets/stylesheets/pictures.scss

```diff
@@ -0,0 +1,37 @@
+// Place all the styles related to the pictures controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+article{
+  background: white;
+  padding: 0;
+  display: block;
+  margin: 70px 0;
+  border: 1px solid #edeeee;
+  border-radius: 3px;
+  header{
+    height: 64px;
+    padding: 14px 20px;
+    a{
+      display: inline-block;
+      padding: 10px 0;
+    }
+  }
+  figure{
+    display: block;
```

Comment:
Properties should be ordered background, background-size, border-bottom, border-top, display, height